src/resource/nonRdfData.ts

```diff
@@ -82,23 +84,27 @@ export async function getFile(
 }
 
 /**
- * Experimental: get a file and it's associated Access Control List.
+ * ```{note} This function is still experimental and subject to change, even in a non-major release.
+ * ```
+ *
+ * Retrieves a file, its resource ACL (Access Control List) if available,
+ * and its fallback ACL from a URL and returns them as a blob.
  *
- * This is an experimental function that gets both a file, the linked ACL Resource (if
- * available), and the ACL that applies to it if the linked ACL Resource is not available. This can
- * result in many HTTP requests being executed, in lieu of the Solid spec mandating servers to
- * provide this info in a single request. Therefore, and because this function is still
- * experimental, prefer [[getFile]] instead.
+ * If the user calling the function does not have access to the file's resource ACL,
+ * [[hasAccessibleAcl]] on the returned blob returns false.
+ * If the user has access to the file's resource ACL but the resource ACL,
+ * [[getResourceAcl]] on the returned blob returns null.
+ * If the fallback ACL is inaccessible by the user,
+ * [[getFallbackAcl]] on the returned blob returns null.
  *
- * If the Resource does not advertise the ACL Resource (because the authenticated user does not have
- * access to it), the `acl` property in the returned value will be null. `acl.resourceAcl` will be
- * undefined if the Resource's linked ACL Resource could not be fetched (because it does not exist),
- * and `acl.fallbackAcl` will be null if the applicable Container's ACL is not accessible to the
```

Comment:
instead of direct references to acl and its properties, I ended up using hasAccessibleAcl/getResourceAcl/getFallbackACL instead.  Let me know, I can put them back or put in addition to.